### PR TITLE
fix: product model, location field updated

### PR DESCRIPTION
**Pull request by cbaksakal**
New location field of product model is as follows:

```
location {
    lat: Number,
    lng: Number
}
```

### CODE REVIEW

File: src/routes/products.js [hunk 1]
```diff
@@ -32,11 +32,25 @@ router.post(
...
+      .isEmpty()
+      .withMessage('Latitude cannot be empty!')
+      .isNumeric()
+      .withMessage('Latitude should be a number!'),
+    body('location.lng')
+      .not()
+      .isEmpty()
+      .withMessage('Longitude cannot be empty!')
+      .isNumeric()
+      .withMessage('Longitude should be a number!'),
```
Comment (muslimomar) [inline src/routes/products.js:53]: Great work utilizing validator for specific object type 🔥

Comment (muslimomar) [reply src/routes/products.js:53]: Let's consider validating the values of `lat` and `lang` too.  Check this [guide](https://stackoverflow.com/questions/7780981/how-to-validate-latitude-and-longitude)

**Review by muslimomar — CHANGES_REQUESTED** [review]
Great work

File: src/routes/products.js (hunk 1 above)
Comment (cbaksakal) [reply src/routes/products.js]: Additional validation for `lat` and `lng` are added.

`(lat >= -90 && lat <= 90)`
`(lng >= -180 && lng <= 180)`

**Review by muslimomar — APPROVED** [review]
Thank you so much for your extreme dedication to the capstone and delivering your tasks in a short notice

**Pull request merged**